Pipfile

```diff
@@ -19,7 +19,7 @@ djangorestframework = "~=3.13"
 filelock = "*"
 fuzzywuzzy = {extras = ["speedup"], version = "*"}
 gunicorn = "*"
-imap-tools = "*"
+imap-tools = "~=0.53.0"
```

Comment:
This is now a version regression compared to `dev` and will break testing there.  

I'm not sure I see a reason to lock this down anyway?  Our testing has improved to catch breaking changes in this area especially

Reply:
But how can you react to breaking changes when these changes are pulled  automatically?
With tests you can detect them, but fixes are only upstream and all older versions break on installation

Reply:
I'm sorry, I'm not following what you mean. 

Nothing is pulled in automatically, dependabot only informs us of new version releases.  And the test suite runs on those PRs to catch issues, besides us reading changelings.

And we've made the changes in dev for 0.54.0, so at least this should match that.

Reply:
Maybe there is a misunderstanding on my side :)

I'll try to explain what I thought happens:

Pipfile is the leading definition from which all dependencies are installed. This file is part of every release and cannot be changed afterwards.
If a wildcard is defined, the latest package Version from the repository is pulled on installation.
Dependabot can monitor requirement files and create alerts on changes. If action is needed, it is taken for the next release.

Therefore old releases break if a wildcard is defined and a breaking change happens.

Reply:
I see.  Thankfully, that won't happen, as the actual versions of dependencies are locked in a couple places `requirements.txt` and `Pipfile.lock`.

`requirements.txt` is used when building the Docker image.  So on a release, which is basically a Git tag named the right way, that tags version of requirements.txt will be baked into the image, and so it can't change.

`Pipfile.lock` is similar, as it is the source of requirements.txt, but includes more detailed information, as well as being useful for local development setup to create an isolated virtualenv.

All that is a long winded way to say the `Pipfile` is the list of dependencies while `requirements.txt` and `Pipfile.lock` lock down the actual versions installed.  So the only way an existing version could change library versions is through a user's manual intervention, and even that pretty much only in a bare metal install.

Thanks for setting this back to `0.54.0`

Reply:
Thanks for explaining :) 
Looks like a lot of thought went into dependency management, I'll leave the decision to you whether to lock down imap-tools in Pipfile or not. Please let me know, I'll change this PR accordingly.